README.md

```
@@ -1,17 +1,20 @@
-# Software Bill of Behavior SBoB 
+# Software Bill of Behavior SBOB 
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)